.github/pull_request_template.md

```diff
@@ -1,25 +1,28 @@
 TODO: Please describe your changes here.
 
-TODO: Delete the following lines but _keep and fill out_ (!) the checklist below.
+<!--
+TODO: Please read the following on ICU Contributing
 
-Thank you for your pull request!\
-General info on contributing: Please see https://github.com/unicode-org/icu/blob/main/CONTRIBUTING.md
+Thank you for your pull request! 
 
-Contributors license agreement (CLA):
-- You will be automatically asked to sign the CLA before the PR is accepted.
-- To sign the CLA: https://cla-assistant.io/unicode-org/icu
+* For general info on contributing: https://github.com/unicode-org/icu/blob/main/CONTRIBUTING.md
```

Comment:
FYI: I had gotten rid of the top-level bullets in https://github.com/unicode-org/icu/pull/3213 probably to visually simplify the instructions. I am ok with bringing the top-level bullets back.